source/getting-started/advanced.markdown

```diff
@@ -14,8 +14,10 @@ Here are some general tutorials on how to setup some of the more advanced deploy
 <div class='advanced-installs-container'>
 <input name='advanced-installs' type='radio' id='upstart-install' checked>
 <input name='advanced-installs' type='radio' id='systemd-install'>
+<input name='advanced-installs' type='radio' id='launchd-install'>
 <label class='menu-selector upstart' for='upstart-install'>Upstart Daemon</label>
 <label class='menu-selector systemd' for='systemd-install'>Systemd Daemon</label>
+<label class='menu-selector launchd' for='launchd-install'>Launchd Daemon</label>
```

Comment:
Yeah, i totally agree, i was just trying to keep with the daemon name pattern. Will do.